apps/aecore/src/aec_governance.erl

```diff
@@ -91,9 +93,14 @@ minimum_gas_price() ->
     0.
 
 %% In key blocks / generations
+%% NOTE: The delay must be at least two generations, otherwise the
+%% payout would be done before anyone had the chance to report the
+%% miner for fraudulent behavior.
 beneficiary_reward_delay() ->
-    aeu_env:user_config_or_env([<<"mining">>, <<"beneficiary_reward_delay">>],
-                               aecore, beneficiary_reward_delay, ?BENEFICIARY_REWARD_DELAY).
+    Delay = aeu_env:user_config_or_env([<<"mining">>, <<"beneficiary_reward_delay">>],
+                                       aecore, beneficiary_reward_delay, ?BENEFICIARY_REWARD_DELAY),
+    [error({beneficiary_reward_delay_too_low, Delay}) || Delay < 2],
```

Comment:
I see this 2 is same 2 as in [`grant_fees`](https://github.com/aeternity/epoch/pull/1557/files#diff-4ac652308bacbca71e5f388759d362c1R825) - a macro would ease readability. Though totally optional.

Reply:
Addressed